test/test.ts

```diff
@@ -875,7 +875,6 @@ describe('gcs-resumable-upload', () => {
 
     it('should set userProject', async () => {
       const scopes = [
-        mockAuthorizeRequest(),
```

Comment:
The request no longer needs to be authorized because it is using a custom endpoint